Code/GraphMol/ScaffoldNetwork/Wrap/testPickleScaffoldNetwork.py

```diff
@@ -0,0 +1,46 @@
+#
```

Comment:
Having this as a separate file is fine, but it could also have been handled using a `skipIf`

Reply:
I thought about that, but then there is the problem of passing the value of the cmake RDK_USE_BOOST_SERIALIZATION variable to the python script. Last time I did something similar, it implied patching TestRunner.py to turn a command line argument into an env var, which was then picked up by the test...

I thought just splitting off the test to a new file was both easier and more proper.

Reply:
Ah, right, there's no particularly easy way to detect on the Python side whether or not the support is there.
I have no objection to having the extra test file in this case.